frontend/src/metabase-types/analytics/event.ts

```diff
@@ -17,4 +17,10 @@ export type CsvUploadClickedEvent = ValidateEvent<{
   triggered_from: "left_nav";
 }>;
 
-export type SimpleEvent = CsvUploadClickedEvent;
+export type NewsletterToggleClickedEvent = {
```

Comment:
@npretto you'd need to call `ValidateEvent` like for `CsvUploadClickedEvent`. You'll see issues immediately.

Reply:
Yeah I rebased what I had quickly without double checking, my bad